libshaderc/include/shaderc.hpp

```diff
@@ -233,7 +235,7 @@ class Compiler {
 
   bool IsValid() const { return compiler_ != nullptr; }
 
-  // Compiles the given source GLSL into a SPIR-V module.
```

Comment:
In a separate CL, we should remove this whole stale-bait blurb and simply say this is like the other `CompileGlslToSpv` but with null options.